proto/beacon/p2p/v1/types.proto

```diff
@@ -145,12 +145,28 @@ message ShardAndCommittee {
 }
 
 message BeaconBlock {
-  repeated bytes ancestor_hashes = 1;
-  uint64 slot = 2;
-  bytes randao_reveal = 3;
-  bytes pow_chain_ref = 4;
-  bytes state_root = 5;
-  google.protobuf.Timestamp timestamp = 6;
-  repeated AggregatedAttestation attestations = 7;
-  repeated SpecialRecord specials = 8;
+  uint64 slot = 1;
+  bytes randao_reveal_hash32 = 2;
+  bytes candidate_pow_receipt_root_hash32 = 3;
+  // Ancestor hashes represent a skip list of ancestor beacon block hashes.
+  // i'th item is the most recent ancestor whose slot is a multiple of 2^i 
+  // for i in the range of 0 to 31.
+  repeated bytes ancestor_hash32s = 4; 
+  bytes state_root_hash32 = 5;
+  // TODO: attestations should be a list of attestionRecords according to spec
```

Comment:
This is fine, I think we renamed `AttestationrRecords` to `AggregatedAttestation`

Reply:
Discussed offline, we will remove aggregatedAttestation in favor of AttestationRecord